src/Entity/PolicyListBuilder.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Drupal\password_enhancements\Entity;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityListBuilder;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Url;
+use Drupal\password_enhancements\Entity\Storage\PolicyEntityStorageInterface;
+use Drupal\user\RoleStorageInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * List builder for password policy config entities.
+ */
+class PolicyListBuilder extends EntityListBuilder {
+
+  /**
+   * Role storage.
+   *
+   * @var \Drupal\user\RoleStorageInterface
+   */
+  protected $roleStorage;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(EntityTypeInterface $entity_type, PolicyEntityStorageInterface $storage, RoleStorageInterface $role_storage) {
+    parent::__construct($entity_type, $storage);
+
+    $this->roleStorage = $role_storage;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function createInstance(ContainerInterface $container, EntityTypeInterface $entity_type): PolicyListBuilder {
+    return new static(
+      $entity_type,
+      $container->get('entity.manager')->getStorage($entity_type->id()),
+      $container->get('entity_type.manager')->getStorage('user_role')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function load(): array {
+    $entity_ids = $this->storage->getQuery()->sort('priority', 'asc')->execute();
+    return $this->storage->loadMultiple($entity_ids);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render(): array {
+    $build = parent::render();
+    $build['table']['#empty'] = $this->t('There is no policy created yet.');
```

Comment:
Unnecessary override? How this is better than what parent sets? ` $this->t('There are no @label yet.', ['@label' => $this->entityType->getPluralLabel()]),`

Reply:
The label of the entity is "Password policy" and I wanted to use only policy here.